### DISCUSSION TITLE
Unable to Open the

### DISCUSSION THREAD

**Discussion by birideslai** [community/community#120070] [Programming Help]
### Select Topic Area

Question

### Body

Hi,
Good morning. I am new to microcontroller programming. I started learing through tutorials. I have my HPC Demo board with PIC18F47Q10 MCU. Unfortunately, I am not able to open the ADC configurator in MCC. Could you help me. The demo code is, (https://github.com/microchip-pic-avr-examples
/
[pic18f47Q10-curiosity-hpc-demo-code]).

**Comment by ebndev** [comment]
Hey @birideslai!

Were you able to figure this out? We don't have many hardware posts in the community so it would be great if I could tag you in future posts if folks want to debug / talk about hardware things in the future!
